Tutorial/Source/HodgkinHuxley.py

```diff
@@ -1,7 +1,9 @@
+from this import d
```

Comment:
Looks good, but what's this for?

Reply:
Sorry! its was a typo. 
Deleted the line and pushed.